xz/FeatureModel.xml

```diff
@@ -1721,11 +1721,11 @@
         <sourceRange>
           <path>src/xz/hardware.c</path>
           <start>
-            <line>21</line>
+            <line>25</line>
             <column>17</column>
           </start>
           <end>
-            <line>21</line>
+            <line>25</line>

```

Comment:
Why did these numbers change? Did you pick a new/other version?

Reply:
Yes, I used the newest commit of xz.

Reply:
Which one did you use?

We are currently defaulting to `4773608554`, in theory we enable the multi version FeatureModel feature. I'm just not sure if that's currently necessary.

Reply:
I also used commit `4773608554`. The locations of this commit don't fit with the locations that are currently used in the FeatureModel, for these few features.

Reply:
Ok, then we should update the lines :+1: